platforms/minigame/platforms/wechat/wrapper/engine/VideoPlayer.js

```diff
@@ -291,7 +291,9 @@ if (cc.internal.VideoPlayer) {
             let self = this;
             let video = this._video;
             if (!video || !this._visible) return;
-
+            if (!this._playing) { // to unify performance of android and ios
+                video.play();
+            }
```

Comment:
Sorry, I don't quite understand the modification. What does `to unify performance of android and ios` mean?

Reply:
on Android, video won't play from begin if called pause() before called stop(). 
But if called play() before stop(), video will play from begin.
I guess this inconsistency is caused by wechat internal api desgin.
detailed description: https://github.com/cocos/3d-tasks/issues/11994#issuecomment-1628347125

Reply:
Got it, so i have two suggestions:
- add more detail information for the comment
- the comment seems wrong, to unify behavior, not performance, right?

Reply:
> add more detail information for the comment

Got it.

> the comment seems wrong, to unify behavior, not performance, right?

Yes. You're right.